packages/perry/src/index.ts

```diff
@@ -1,13 +1,27 @@
-import IPerryOptions from "@/interfaces/IPerryOptions";
-import IPerryReportInfo from "@/interfaces/IPerryReportInfo";
-import IPerryScreenRecorder from "@/interfaces/IPerryScreenRecorder";
+/** Widget Component Props interface */
+import {
+  IPerryOptions,
+  IPerryReportInfo,
+  IScreenRecorder as IPerryScreenRecorder,
```

Comment:
We can expose it as IPerryScreenRecorder from the `perry-interfaces` just to avoid the noise of renaming it around code =)